src/gstools/krige/base.py

```diff
@@ -39,6 +43,40 @@
 """dict: Standard pseudo-inverse routines"""
 
 
+def _calc_field_krige(krig_mat, krig_vecs, cond, num_threads=None):
```

Comment:
There is an issue in krigesum.pyx L63, where `num_threads` is defined as follows:
```python
const int num_threads=1
```
This should be just
```python
num_threads=None
```

Reply:
Also a great catch, thanks!